### DISCUSSION TITLE
Any plans to support Azure?

### DISCUSSION THREAD

**Discussion by nikolay** [le0pard/certonid#1] [Q&A]

**Comment by le0pard** [comment]
Yep. Main question - is Azure support Golang or at least binary execution? This document says "no": https://docs.microsoft.com/en-us/azure/azure-functions/supported-languages

**Comment by le0pard** [comment]
But looks like it is possible: https://github.com/Azure/azure-functions-host/wiki/Language-Extensibility I will add this in todo
